### PR TITLE
remove doubles before save

### PR DESCRIPTION
**Pull request by DK101010**
### Description
Currently it is possible to add the same tag several times. This PR remove the doubles for api and ui input.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### How Has This Been Tested?

In own Vmware env.

### CODE REVIEW

File: server/src/main/java/com/cloud/resource/ResourceManagerImpl.java [hunk 1]
```diff
@@ -1654,13 +1655,12 @@ public Host updateHost(final UpdateHostCmd cmd) throws NoTransitionException {
                 }
             }
         }
-
-        final List<String> hostTags = cmd.getHostTags();
+        List<String> hostTags = cmd.getHostTags();
```
Comment (weizhouapache) [inline server/src/main/java/com/cloud/resource/ResourceManagerImpl.java:1658]: @DK101010 why removed 'final' ?

Comment (DK101010) [reply server/src/main/java/com/cloud/resource/ResourceManagerImpl.java:1658]: Arrrr... my bad, I removed this in my first version. but then I found a better way and forgot the add this again.

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Review by rhtyd — APPROVED** [review]

**Comment by rhtyd** [comment]
(rekicked travis)

**Review by weizhouapache — APPROVED** [review]
tested ok.

**Comment by sureshanaparti** [comment]
@blueorangutan test

**Review by nvazquez — APPROVED** [review]

**Pull request merged**